ddp-workspace/projects/ddp-rgp/src/app/components/study-messages/study-messages.component.html

```diff
@@ -1,96 +1,104 @@
-        class="app-table__cell"
-        mat-cell
-        *matCellDef="let message"
-        [attr.label]="
-          'DashboardPage.StudyMessages.Header.Description' | translate
-        "
-      >
-        <div class="app-table__cell--description">
-          <p [innerHTML]="message.message | translate" style="margin: 0"></p>
+        <ng-container matColumnDef="description">
+          <th
+            class="
+              app-table__cell app-table__cell--header app-table__col--description
+            "
+            mat-header-cell
+            *matHeaderCellDef
+          >
+            {{ 'DashboardPage.StudyMessages.Header.Description' | translate }}
+          </th>
 
-          <div *ngIf="hasMoreText(message.more) | async">
-            <div
-              *ngIf="!!showMoreMap[message.more]"
-              [innerHTML]="message.more | translate"
-              class="show-more-container"
-            ></div>
+          <td
+            class="app-table__cell"
+            mat-cell
+            *matCellDef="let message"
+            [attr.label]="
+              'DashboardPage.StudyMessages.Header.Description' | translate
+            "
+          >
+            <div class="app-table__cell--description">
+              <p [innerHTML]="message.message | translate" style="margin: 0"></p>
 
-            <button class="show-more-btn" (click)="toggleShowMore(message)">
-              {{
-                'DashboardPage.StudyMessages.Actions.' +
-                  (!!showMoreMap[message.more] ? 'ShowLess' : 'ShowMore')
-                  | translate
-              }}
-            </button>
-          </div>
-        </div>
-      </td>
-    </ng-container>
+              <div *ngIf="hasMoreText(message.more) | async">
+                <div
+                  *ngIf="!!showMoreMap[message.more]"
+                  [innerHTML]="message.more | translate"
+                  class="show-more-container"
+                ></div>
 
-    <tr
-      class="app-table__row app-table__row--header"
-      mat-header-row
-      *matHeaderRowDef="displayedColumns"
-    ></tr>
-    <tr
-      class="app-table__row"
-      mat-row
-      *matRowDef="let row; columns: displayedColumns"
-    ></tr>
-  </table>
+                <button class="show-more-btn" (click)="toggleShowMore(message)">
+                  {{
+                    'DashboardPage.StudyMessages.Actions.' +
+                      (!!showMoreMap[message.more] ? 'ShowLess' : 'ShowMore')
+                      | translate
+                  }}
+                </button>
+              </div>
+            </div>
+          </td>
+        </ng-container>
+
+        <tr
+          class="app-table__row app-table__row--header"
+          mat-header-row
+          *matHeaderRowDef="displayedColumns"
+        ></tr>
+        <tr
+          class="app-table__row"
+          mat-row
+          *matRowDef="let row; columns: displayedColumns"
+        ></tr>
+      </table>
+    </div>
+  </ng-container>
```

Comment:
Most of the change here is indentation to wrap the `<table>` inside the containers. New stuff is just the `<h4>` name and the for-loop.